lib/sonos.js

```diff
@@ -704,18 +719,23 @@ Sonos.prototype.playTuneinRadio = function (stationId, stationTitle, callback) {
 }
 
 /**
- * Add a song from spotify to the queue
+ * Add a song from spotify to the queue, just a shortcut to sonos.queue('spotify:track:....')
  * @param  {String}   trackId      The spotify track ID
  * @param  {Function} callback (err, success)
  */
-Sonos.prototype.addSpotify = function (trackId, callback) {
-  var uri = 'x-sonos-spotify:spotify%3atrack%3a' + trackId
-  var meta = '<DIDL-Lite xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:upnp="urn:schemas-upnp-org:metadata-1-0/upnp/" xmlns:r="urn:schemas-rinconnetworks-com:metadata-1-0/" xmlns="urn:schemas-upnp-org:metadata-1-0/DIDL-Lite/"><item id="00032020spotify%3atrack%3a' + trackId + '" restricted="true"><dc:title></dc:title><upnp:class>object.item.audioItem.musicTrack</upnp:class><desc id="cdudn" nameSpace="urn:schemas-rinconnetworks-com:metadata-1-0/">SA_RINCON' + this.options.spotify.region + '_X_#Svc' + this.options.spotify.region + '-0-Token</desc></item></DIDL-Lite>'
+Sonos.prototype.addSpotifyTrack = function (trackId, callback) {
```

Comment:
Worth taking this op to rename to `queueSpotifyTrack`? Think it's more descriptive or remove altogether and just support spotify tracks URIs via the `queue()`/`play()` functions.

Reply:
I think removing them would be great. since they are only shortcuts at the moment.....